Automattic/wp-calypso
# Tailored Flows: Remove Free Setup Flow

Related to:
- https://github.com/Automattic/dotcom-forge/issues/9333

## Proposed Changes

- Removed the Free Setup tailored flow.
- Removed related components.

## Why are these changes being made?


- We need to remove the code of deprecated flows.


## Testing Instructions



- Go to /setup/free
- Make sure it still redirects to /start/free
- Check for console errors.
- Check for regressions.
- Complete the /start/free flow and make sure it works as expected.

## Pre-merge Checklist
